pages/en/contrib/doc-contrib.md

```diff
@@ -25,13 +25,13 @@ Follow these steps:
 1. Click on **Edit this page** (top of page) to fork the repo.
 1. Edit the page or pages as needed.
 1. Commit to your fork/branch.
-1. Open a PR for your changes.  If there is an associated issue, please reference it in your PR.
+1. Open a PR for your changes.  If there is an associated issue, please reference it in your PR. The tutorial [Submitting a pull request to LoopBack 4](https://loopback.io/doc/en/lb4/submitting_a_pr.html) contains expert and beginner instructions to help a contributor with the pull request flow.
```

Comment:
Maybe it's better to use a positive style, such as: `Check out [submitting a pull request to LoopBack 4](https://loopback.io/doc/en/lb4/submitting_a_pr.html) for detailed instructions.`

Reply:
done